tests/foundry/foundry.k.check.expected

```diff
@@ -12,14 +12,14 @@ module BLOCKPARAMSTEST-BIN-RUNTIME
       
     
     syntax Field ::= BlockParamsTestField
     
     syntax BlockParamsTestField ::= "IS_TEST" [symbol(), klabel(field_BlockParamsTest_IS_TEST)]
     
-    syntax BlockParamsTestField ::= "_failed" [symbol(), klabel(field_BlockParamsTest__failed)]
+    syntax BlockParamsTestField ::= "failed" [symbol(), klabel(field_BlockParamsTest_failed)]
     
```

Comment:
This diff, I am not confident is correct. We need to investigate _why_ this happened.